src/shared/styles/fonts.js

```diff
@@ -0,0 +1,22 @@
+import { Platform } from 'react-native';
```

Comment:
I'd probably rename this file to `typography.js`. I think fonts should only be used to address the font files themselves. What do you think?

Reply:
@satazor, what do you think?

Reply:
I've made the change to `typography.js` for the time being. We can return to this topic later if necessary.